Mathlib/Algebra/Algebra/NonUnitalSubalgebra.lean

```diff
@@ -76,6 +76,11 @@ instance instNonUnitalSubsemiringClass : NonUnitalSubsemiringClass (NonUnitalSub
 instance instSMulMemClass : SMulMemClass (NonUnitalSubalgebra R A) R A where
   smul_mem := @fun s => s.smul_mem'
 
+instance instNonUnitalSubringClass {R : Type u} {A : Type v} [CommRing R] [NonUnitalNonAssocRing A]
```

Comment:
This should be deleted, it's leftover from a tricky merge

Reply:
Done.